### PR TITLE
[DYN-7833] Run All should be disabled in Custom Nodes

### PR DESCRIPTION
**Pull request by ivaylo-matov**
PR aims to address [DYN-7833](https://jira.autodesk.com/browse/DYN-7833) and the conversation in this Slack [tread](https://autodesk.slack.com/archives/CPJ5UQW0Z/p1731570085516989)

The "Run All" button is now disabled when the workspace is in Custom Node mode. The button is greyed out and a tooltip is displayed to explain that the button is not available in this mode.

DYN-7833-Fix

@QilongTang 
@reddyashish 
@dnenov

### CODE REVIEW

**Comment by QilongTang** [comment]
Thank you @ivaylo-matov I am curious with this code, if user is simply switching back and forth between homeworkspace and custome node workspace, would the execution time data come back after switching from custom node workspace?

**Comment by ivaylo-matov** [comment]
@QilongTang , right now it does not.
So, do we want to display default UI in Custom Node mode and then bring back latest result once we are back in the main graph?

**Comment by ivaylo-matov** [comment]
aware of the failed build. Will check once I come back...

File: TuneUp/TuneUpWindowViewModel.cs [hunk 1]
```diff
@@ -1013,20 +1026,143 @@ private void CurrentWorkspaceModel_GroupRemoved(AnnotationModel group)
...
         #endregion        
 
+        #region Cache
+
+        /// <summary>
+        /// Represents cached profiling data for a workspace. Includes node collections and execution times.
+        /// </summary>
+        private class WorkspaceProfilingData
+        {
+            public Guid GraphGuid { get; set; }
```
Comment (reddyashish) [inline TuneUp/TuneUpWindowViewModel.cs:1080]: Add summary to these public properties.

**Review by reddyashish — APPROVED** [review]
LGTM. Will merge once the checks pass.

**Pull request merged**